packages/validator/src/services/prepareBeaconProposer.ts

```diff
@@ -37,13 +38,19 @@ export function pollPrepareBeaconProposer(
     const indicesChunks = batchItems(validatorStore.getAllLocalIndices(), {batchSize: REGISTRATION_CHUNK_SIZE});
 
     for (const indices of indicesChunks) {
+      const proposers: ProposerPreparationData[] = [];
       try {
-        const proposers = indices.map(
-          (index): routes.validator.ProposerPreparationData => ({
+        indices.forEach((index) => {
+          const feeRecipient = validatorStore.getFeeRecipientByIndex(index);
+          const pubKey = validatorStore.getPubkeyOfIndex(index);
+
+          proposers.push({
             validatorIndex: String(index as number),
-            feeRecipient: validatorStore.getFeeRecipientByIndex(index),
-          })
-        );
+            feeRecipient,
+          });
+
+          logger.info("Validator exists in Beacon Chain", {validatorIndex: index, feeRecipient, pubKey});
```

Comment:
to be considered if `Beacon Chain` should use upper case, based on current logs we rarely/never do this

```suggestion
          logger.info("Validator exists in beacon chain", {validatorIndex: index, feeRecipient, pubKey});
```